README.md

```diff
@@ -213,6 +213,39 @@ spec:
 
 > Note: pay attention to the name of the selector above. You will need to use it to bind the identity to your pod.
 
+If you use the Helm Chart to deploy the custom metrics adapter to your Kubernetes cluster, you can configure Azure AD Pod Identity directly in the config values:
+
+```yaml
+azureAuthentication:
+  # method: {msi,clientSecret,clientCertificate,aadPodIdentity}
+  method: msi
```

Comment:
I think it would make more sense to set this to `aadPodIdentity` and remove the `client` settings and fill in an example for the settings needed for aad-pod-idenity.  It was a bit confusing to see all SP options under aad-pod-identity section.

Reply:
Do you want to enable pod identity by default ? I was not sure about that as aad-pod-identity requires more infrastructure running into the cluster that just using MSI or SP. And it's also specific if you want that each pod in the cluster is able to get its own identity

Reply:
No, But for this section of the readme that is showing how to use Pod Identity I think it makes sense to drop out the other options?

Reply:
I've done the update